bundles/org.openhab.binding.ecovacs/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="ecovacs" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Ecovacs Binding</name>
+	<description>This is the binding for Ecovacs Deebot vacuum cleaners.</description>
```

Comment:
This could also be added:
```suggestion
	<description>This is the binding for Ecovacs Deebot vacuum cleaners.</description>
	<connection>cloud</connection>
```